cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CloudProviderDTO.java

```diff
@@ -145,11 +148,19 @@ public void setSupportedOperatingSystems( List<OSType> supportedOperatingSystems
         this.supportedOperatingSystems = supportedOperatingSystems;
     }
 
+    /**
+     * @deprecated use {@link InstancePricingDTO#getReservedCloudPricing()}
+     */
+    @Deprecated
     public Map<Integer, BigDecimal> getDiscounts()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)